style.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&family=Syne+Mono&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: bold;
+}
+body{
+    background: #736453;
```

Comment:
remove this line because it doesn't affect on the style( the below line overide it)